website/i18n/en/docusaurus-plugin-content-docs/current/concepts/app-splitting.md

```diff
@@ -75,8 +76,13 @@ The second level of separation is by **specific BL functionality**
 |   ├── sign-up
 |   ├── feed
 |   └── ...
+├── widgets/
+|   # Slices for the implementation of independent page blocks
```

Comment:
"Slices implementing independent page blocks"

Also, check grammar in similar cases.